annotations/builder/src/main/java/io/sundr/builder/internal/functions/ToMethod.java

```diff
@@ -137,17 +144,21 @@ public Method apply(Property property) {
                 } else if (IS_LIST.apply(type) || IS_SET.apply(type)) {
                     statements.add(new StringStatement("if (this." + fieldName + " == null) { this." + fieldName + " = " + property.getAttribute(LAZY_INIT) + ";} else {_visitables.removeAll(this."+fieldName+"); this." + fieldName + ".clear();}"));
                     String addToMethodName = "addTo" + property.getNameCapitalized();
-                    statements.add(new StringStatement("if (" + argumentName + " != null) {for (" + unwraped.toString() + " item : " + argumentName + "){this." + addToMethodName + "(item);}} return (" + returnType + ") this;"));
+                    statements.add(new StringStatement("if (" + argumentName + " != null) {for (" + unwrapped.toString() + " item : " + argumentName + "){this." + addToMethodName + "(item);}} return (" + returnType + ") this;"));
                 }
                 return statements;
-            } else if (isBuildable(unwraped) && !isAbstract(unwraped)) {
```

Comment:
Did u hit any issues and u decided to turn `else if ` into a plain `if`?
How would this code work, if u had to deal with a `List` of `Buildable` items?

Reply:
Each if else block had a return statement in so the else bit is redundant - will return or drop through. Just made it more readable, no problems.

Reply:
What I mean is:

```java
if (...) {
     return ...;
} else if (...) {
     return ...;
} else if (...) {
     return ...;
} else {
     return ...;
}
```

is the same functionally without the redundant elses as:

```java
if (...) {
     return ...;
}
if (...) {
     return ...;
}
if (...) {
     return ...;
}

return ...;
```

Reply:
Right! Didn't realize there was a return statement.